<launch>
    <param name="num_cams" value="1"/>
    <node pkg="roboteam_world" type="filtered_world" name="world"/>
    <node pkg="roboteam_vision" type="roboteam_vision" name="vision"/>
    <node pkg="roboteam_steering" type="steering" name="steering" output="screen"/>
    <node pkg="roboteam_robothub" type="roboteam_robothub" name="sender"/>
</launch>
